### PR TITLE
Feature: csvsql: specify --query multiple times #1160

### PR DESCRIPTION
**Pull request by badbunnyyy**
Hi jpmckinney,

I hope this is what you need. Thank you for your effort.

Best,
Stefan/ badbunnyyy

### CODE REVIEW

File: csvkit/utilities/csvsql.py [hunk 1]
```diff
@@ -221,20 +221,20 @@ def _failsafe_main(self):
...
                 # Execute the specified SQL queries.
-                queries = query.split(';')
                 rows = None
 
-                for q in queries:
-                    if q.strip():
-                        rows = self.connection.execute(q)
+                for query in queries:
+                    if query.strip():
+                        rows = self.connection.execute(query)
```
Comment (slhck) [inline csvkit/utilities/csvsql.py:237]: This means that `rows` will be set to whatever the last query returns, right? Wouldn't it be better to store all the rows from all queries in a (flat) list?

**Review by slhck — COMMENTED** [review]
This is a nice proposal. Too bad there has been no feedback yet.

File: csvkit/utilities/csvsql.py (hunk 1 above)
Comment (jpmckinney) [reply csvkit/utilities/csvsql.py:237]: Yes, that is the existing functionality, so not a problem with the PR.

**Review by jpmckinney — APPROVED** [review]

**Comment by jpmckinney** [comment]
Thank you @badbunnyyy !

**Pull request merged**